### ISSUE TITLE
Unable to install plugin

### ISSUE THREAD

**Issue by blaskowski94** [realestate-com-au/json-bourne-sfdx-cli#5]
#### Issue Description

Whenever I try to install the plugin the installation fails. From the documentation I was not sure whether this was intended to be installed as a standalone npm package or an sfdx plugin. I tried both.

`sfdx plugins:install json-bourne-sfdx` output:
![Screen Shot 2019-11-26 at 10 40 35 AM](https://user-images.githubusercontent.com/15270286/69653819-614f7900-1039-11ea-8d19-f457434633c5.png)

` yarn global add json-bourne-sfdx` output:
![Screen Shot 2019-11-26 at 10 41 12 AM](https://user-images.githubusercontent.com/15270286/69653820-614f7900-1039-11ea-94ef-dab71246c036.png)

`npm install -g json-bourne-sfdx` output: 
![Screen Shot 2019-11-26 at 10 41 31 AM](https://user-images.githubusercontent.com/15270286/69653821-614f7900-1039-11ea-88c7-a3601e0ff5a2.png)

sfdx (core) version: 7.33.2
npm version: 6.13.0
yarn version: 1.19.2
node version: 10.16.3
operating system: macOS 10.14.4

#### Issue Fix Ideas

Not too sure. Seems like a dependency issue, although I didn't see any dependencies that directly rely on the package causing the error. Not sure where that is coming from. 
 
#### Helpful Links

Top google result for error: https://github.com/substack/shoe/issues/36

I heard about this from the talk you gave at Dreamforce last week, I really like the idea. Thanks for sharing.

**Comment by aimaj** [comment]
Hi Bob,

I was able to reproduce your issue with my versions being:
npm version: 6.13.1
node version: v8.11.1
sfdx version: 7.33.2

From the sfdx plugins install output, looks like the 'verbose' dependency might be related to the issue, will investigate further

**Comment by blaskowski94** [comment]
Ah, I was able to see the following dependency chain: 

verbose > reconnect > sockjs-stream > sockjs-client

Looks like sockjs-stream depends on a specific branch of a fork of the main sockjs-stream repo. There is an issue open in that repo about this: https://github.com/Raynos/sockjs-stream/issues/3.  It's from 2016 so not likely to be fixed. 

I also noticed that verbose and the dependencies in this chain have all not been updated in 5 years or more and some of the dependent repos have been archived. 

Is the verbose dependency actually being used anywhere? I don't see any actual references to it in the code.
